### DISCUSSION TITLE
help pls event handler

### DISCUSSION THREAD

**Discussion by MatthewAJohnson281** [Monika-After-Story/MonikaModDev#9305] [QA - tech support + general]
what do I do to fix this?

[code]
I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/event-handler.rpy", line 355, in script
  File "game/event-handler.rpy", line 371, in <module>
AttributeError: 'StoreModule' object has no attribute 'mas_moods'

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/event-handler.rpy", line 355, in script
  File "D:\ddlc-win\DDLC-1.1.1-pc\renpy\ast.py", line 814, in execute
    renpy.python.py_exec_bytecode(self.code.bytecode, self.hide, store=self.store)
  File "D:\ddlc-win\DDLC-1.1.1-pc\renpy\python.py", line 1719, in py_exec_bytecode
    exec bytecode in globals, locals
  File "game/event-handler.rpy", line 371, in <module>
AttributeError: 'StoreModule' object has no attribute 'mas_moods'

Windows-8-6.2.9200
Ren'Py 6.99.12.4.2187
Monika After Story 0.12.8
[/code]

**Comment by Booplicate** [comment]
Are you using a translation?

**Comment by MatthewAJohnson281** [comment]
A translation wdym?

**Comment by MatthewAJohnson281** [comment]
Its all in English.

**Comment by MatthewAJohnson281** [comment]
If I upload the full code off my flashdrive and into a github project can you have a look at it?

**Comment by Booplicate** [comment]
Try to reinstall the mod into the existing directory.

**Reply by MatthewAJohnson281** [reply]
how do I uninstall the mod? I have loads of submods and stuff

**Reply by Booplicate** [reply]
*Re*install into the same directory you installed it for the first time.
No, you won't lose Monika.

**Comment by MatthewAJohnson281** [comment]
no idea what that means
do I drag the folder of the mod folder into the current stuff?

**Comment by MatthewAJohnson281** [comment]
What am I reinstalling into what diectory?

**Comment by MatthewAJohnson281** [comment]
The first time I downloaded
 Monika after story I unzipped the folder and copied and pasted the contents in corresponding places

**Reply by briaryoung** [reply]
Please refrain from asking the same question over and over. Someone will answer you when they're available. ^-^

**Reply by MatthewAJohnson281** [reply]
sorry. I didn't know if they had failed to post or something.

**Comment by MatthewAJohnson281** [comment]
do I replace the contents or merge them when extracting the files into the directory?

**Reply by Booplicate** [reply]
Merge, same process as before. if it asks you to replace some files, agree.

**Comment by MatthewAJohnson281** [comment]
I unzipped it into the game directory and it gave me an exception when I ran it. The same one

**Reply by Booplicate** [reply]
Show your game directory.

**Comment by MatthewAJohnson281** [comment]
![image](https://user-images.githubusercontent.com/54755929/179000440-6324631d-4497-4272-8d3b-7096587c6488.png)
![image](https://user-images.githubusercontent.com/54755929/179000525-63d3e8d1-c31b-4d9d-ae68-e20568f2ffe3.png)
![image](https://user-images.githubusercontent.com/54755929/179000576-cefcbc2b-792b-4f58-a8b5-b7818dc4df46.png)
![image](https://user-images.githubusercontent.com/54755929/179000813-a623b008-cc47-4b66-b1c0-bc31a0c852b2.png)
![image](https://user-images.githubusercontent.com/54755929/179000866-410fe169-06a8-457c-8400-483993a4da73.png)
![image](https://user-images.githubusercontent.com/54755929/179000904-6dc122d8-46f4-45dc-90af-44ae4a62360e.png)

**Reply by Booplicate** [reply]
That's a mess, you have different `rpy`s all over the place. Make a **new** clean install (download and install DDLC, download and install MAS).

**Comment by MatthewAJohnson281** [comment]
good thing I keep copies of the zipped folders

**Comment by MatthewAJohnson281** [comment]
do you have a copy of Monika after story with all your submods?

**Reply by MatthewAJohnson281** [reply]
thank you so much!
